resource-service/src/component-tests/java/uk/gov/gchq/palisade/component/resource/repository/ReactivePersistenceLayerTest.java

```diff
@@ -113,23 +113,26 @@ void testAddAndGetReturnsResource() {
         // Given the setup
 
         // When getting a resource from the persistence layer by resourceId
-        final List<LeafResource> idResult = persistenceLayer.getResourcesById(resource.getId()).orElseThrow()
+        var idResult = persistenceLayer.getResourcesById(resource.getId())
```

Comment:
Small point - on this line you use a var and in the next section you don't.  Should be either one convention or the other.

Reply:
Will probably opt for using the `var` instance for these as they are in the test class.